src/main/java/uk/gov/hmcts/cmc/claimstore/services/OffersService.java

```diff
@@ -46,16 +46,15 @@ public void makeOffer(Claim claim, Offer offer, MadeBy party) {
         eventProducer.createOfferMadeEvent(claim);
     }
 
-    @Transactional
     public void accept(Claim claim, MadeBy party) {
         assertSettlementIsNotReached(claim);
 
         Settlement settlement = claim.getSettlement()
             .orElseThrow(() -> new ConflictException("Offer has not been made yet."));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Offer has not been made yet." 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1192)